Source/inv.cpp

```diff
@@ -1478,39 +1478,24 @@ static void CheckBookLevel(PlayerStruct &player)
 static void CheckNaKrulNotes(PlayerStruct &player)
 {
 	int idx = player.HoldItem.IDidx;
+	_item_indexes notes[] = { IDI_NOTE1, IDI_NOTE2, IDI_NOTE3 };

```

Comment:
This can probably be `static`/`constexpr`

Reply:
I'm not an expert when it comes to static/constexpr but I don't see anything that could be evaluated at compile time since idx is dynamic 🤷

Reply:
While `idx` is, the list itself is static, so it should work just fine.